### PR TITLE
Xarray example

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
Add an xarray example

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How has this been tested?

- [x] `example/xarray_nD_image.py`

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sofroniewn** [comment]
image

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
@sofroniewn I've changed the imports to import napari rather than using from napari import *, but approved. Just waiting for CI before merging.

**Pull request merged**
